src/ucp/core/ucp_context.c

```diff
@@ -295,7 +295,11 @@ static ucs_config_field_t ucp_config_table[] = {
    "Experimental: enable new protocol selection logic",
    ucs_offsetof(ucp_config_t, ctx.proto_enable), UCS_CONFIG_TYPE_BOOL},
 
-  {NULL}
+  {"PROTO_INDIRECT_ID", "auto",
+   "Enable indirect IDs to object pointers in wire protocols",
```

Comment:
"Enable indirect IDs to object pointers (endpoint, request) in wire protocols. A value of 'auto' means to enable only if error handling is enabled on the endpoint."